airbyte-integrations/connectors/source-cart/acceptance-test-config.yml

```diff
@@ -8,23 +8,21 @@ tests:
     - config_path: "secrets/config.json"
       status: "succeed"
     - config_path: "integration_tests/invalid_config.json"
-      status: "exception"
+      status: "failed"
+      timeout_seconds: 180
```

Comment:
180sec for a simple check is too much, I guess, what is the error we got here?

Reply:
the default behavior of read_records is to make 5 retries, and only then we can catch the external exception. If the user make a syntax mistake in store domain name, it will try to connect to unexisting website. 5 retries 5 sec, 10 sec etc totally have the delay a bit less then 180 seconds - Without changing read_records default behavior.

If it will be token mistake, the response will be fast, 180 is to cover wrong store domain name